util.go

```diff
@@ -25,10 +35,19 @@ func fileExists(fname string) bool {
 	return true
 }
 
+func isFileAccessible(fname string) bool {
+	return  fileExists(fname) && isFileReadable(fname)
```

Comment:
There is an extra whitespace. I suggest running `gofmt` in the source code directory. This should ensure the formatting is correct. If you use `vim` as an editor, there's an fantastic plugin for Go called [vim-go](https://github.com/fatih/vim-go) that does this (among other awesomeness) every time when you save the file.